### PR TITLE
chore(doc): add steps to create cstor pool using local devices

### PR DESCRIPTION
**Pull request by shovanmaity**
add steps to create cStor pool using local bd

Signed-off-by: Shovan Maity <shovan.maity@mayadata.io>

### CODE REVIEW

File: demo/local-device/readme.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+### Prerequisite
```
Comment (AmitKumarDas) [inline demo/local-device/readme.md:1]: can the readme be named as README

File: demo/local-device/readme.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  name: mirror-mysql-pool
+  namespace: openebs
+spec:
+  diskConfig:
+    local:
+      blockDeviceSelector:
+        selectorTerms:
+        - matchLabels:
+            mirror-pool: mysql
+            # Selected block devices should labeled with `mirror-pool=mysql`
```
Comment (AmitKumarDas) [inline demo/local-device/readme.md:126]: can we put this comment before blockDeviceSelector

File: demo/local-device/readme.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  name: cstorpoolauto
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - cstorclusterconfigs
+  - blockdevices
+  - cstorpoolclusters
+  - nodes
+  verbs:
```
Comment (AmitKumarDas) [inline demo/local-device/readme.md:33]: can we provide appropriate verbs per resources?
Atleast mention the same as comments. 
We can come back with granular control in a later PR

File: demo/local-device/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
+## Introduction
+CSPC operator which is a component of OpenEBS. Its a alpha feature, and new version of existing [SPC](https://docs.openebs.io/docs/next/ugcstor.html#creating-cStor-storage-pools).
```
Comment (vishnuitta) [inline demo/local-device/README.md:2]: CSPC operator which is a component of OpenEBS, is a alpha feature and

**Pull request merged**